pod/perlfunc.pod

```diff
@@ -6543,32 +6543,60 @@ X<rand> X<random>
 
 =for Pod::Functions retrieve the next pseudorandom number
 
-Returns a random fractional number greater than or equal to C<0> and less
+Returns a random fractional number greater than or equal to C<0> and I<less>
 than the value of EXPR.  (EXPR should be positive.)  If EXPR is
-omitted, the value C<1> is used.  Currently EXPR with the value C<0> is
-also special-cased as C<1> (this was undocumented before Perl 5.8.0
-and is subject to change in future versions of Perl).  Automatically calls
-L<C<srand>|/srand EXPR> unless L<C<srand>|/srand EXPR> has already been
-called.  See also L<C<srand>|/srand EXPR>.
+omitted, the value C<1> is used.
 
-Apply L<C<int>|/int EXPR> to the value returned by L<C<rand>|/rand EXPR>
-if you want random integers instead of random fractional numbers.  For
-example,
+    my $num1 = rand();        # Random float between 0 and 1
+    my $num2 = rand(10);      # Random float between 0 and 10
+    my $num3 = int(rand(10)); # Random integer between 0 and 10
 
-    int(rand(10))
+B<Notes:>
 
-returns a random integer between C<0> and C<9>, inclusive.
+=over
+
+=item *
+
+Calling C<rand> Automatically calls L<C<srand>|/srand EXPR> unless
+srand has already been called.
+
+=item *
+
+Currently EXPR with the value C<0> is
+also special-cased as C<1>. This was undocumented before Perl 5.8.0
+and is subject to change in future versions of Perl.
```

Comment:
The word "also" here seems to be left over from the previous version. I'd remove it, and just say "0 is special-cased to mean 1".
Also this paragraph needs reflowing.

Reply:
Good catch... this is corrected.